src/components/VerticalList.js

```diff
@@ -88,10 +88,6 @@ export const VerticalList = ({
             );
           }
         } else if (isLoading) {
-          if (ListFooterLoadingIndicator) {
-            return <ListFooterLoadingIndicator />;
-          }
-
```

Comment:
why was this removed? there can be `ListFooterLoadingIndicator` passed for the vertical list. i can't see any infos about this removal in the commit message, right?

Reply:
The reason I deleted it is that two loading indicators appear on the screen. But now it's working correctly in my tests. I'm adding this back.

Reply:
https://github.com/smart-village-solutions/smart-village-app-app/pull/1036/commits/5f5bc75c31968cd9f42e97b005ff6210b97bf327